### PR TITLE
Adds the performance dashboard back

### PR DESCRIPTION
**Pull request by megelatim**
The performance dashboard was deleted in a previous commit.  This brings the performance dashboard back and updates the README for both of the dashboards.

The performance dashboard is still used and referenced in the Google Cloud SAP docs (https://cloud.google.com/solutions/sap/docs/sap-hana-monitoring-agent-user-guide#sap-hana-monitoring-dashboard)

### CODE REVIEW

**Comment by megelatim** [comment]
Requesting review for this

File: dashboards/sap-hana/metadata.yaml [hunk 1]
```diff
@@ -8,4 +8,12 @@ sample_dashboards:
...
\ No newline at end of file
+        platform: GCE
+  -
+    category: SAP HANA
+    id: performance
+    display_name: SAP HANA Performance Monitoring
+    description: "This dashboard has 8 charts to display the metrics ingested via the HANA Monitoring Agent SAP HANA V2.0. The metrics include Instance Memory, Schema Records, Connections, and Rowstore Memory Size etc."
+    related_integrations:
+      - id: saphana
+        platform: GCE
```
Comment (EvanSimpson) [inline dashboards/sap-hana/metadata.yaml:19]: This is for dashboards with metrics collected by the Ops Agent, please remove.

File: dashboards/sap-hana/README.md [hunk 2]
```diff
@@ -1,11 +1,18 @@
 ### Dashboards for SAP HANA
 
-#### Notes
+|SAP HANA GCE Overview|
+|:------------------|
+|Filename: [sap-hana-gce-overview.json](sap-hana-gce-overview.json)|
+|This dashboard has charts displaying `Instance Memory`, `Service Memory Used`, `Schema Estimated Max Memory`, `Rowstore Memory Size`, `Schema Read Outliers`,`Schema Write Outlines`, `Schema Record Count Outliers`, `System Connections`, `Alerts`, and `Blocked Transactions` from SAP HANA as well as charts of infrastructure related metrics for the running SAP HANA VMs: `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`. There is also a card with links to docs and the SAP HANA logs in Cloud Logging.|
 
+#### Notes
```
Comment (EvanSimpson) [inline dashboards/sap-hana/README.md:8]: Please move the notes header back to the top of the file. You can remove the notes content with the suggested description change above.

File: dashboards/sap-hana/README.md [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
-|SAP HANA Overview|
+|SAP HANA Performance Monitoring|
 |:------------------|
-|Filename: [sap-hana-gce-overview.json](sap-hana-gce-overview.json)|
-|This dashboard has charts displaying `Instance Memory`, `Service Memory Used`, `Schema Estimated Max Memory`, `Rowstore Memory Size`, `Schema Read Outliers`,`Schema Write Outlines`, `Schema Record Count Outliers`, `System Connections`, `Alerts`, and `Blocked Transactions` from SAP HANA as well as charts of infrastructure related metrics for the running SAP HANA VMs: `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`. There is also a card with links to docs and the SAP HANA logs in Cloud Logging.|
+|Filename: [performance.json](performance.json)|
+|This dashboard has 8 charts to display the metrics ingested via the HANA Monitoring Agent. The metrics include `Instance Memory`, `Schema Records`, `Connections`, and `Rowstore Memory Size` etc.|
+
+#### Notes
+- This dashboard is based on Google's [Monitoring agent for SAP HANA V2.0](https://cloud.google.com/solutions/sap/docs/sap-hana-monitoring-agent-planning-guide)
```
Comment (EvanSimpson) [inline dashboards/sap-hana/README.md:18]: Please remove.

File: dashboards/sap-hana/README.md [hunk 4]
```diff
@@ -1,11 +1,18 @@
 ### Dashboards for SAP HANA
 
-#### Notes
+|SAP HANA GCE Overview|
+|:------------------|
+|Filename: [sap-hana-gce-overview.json](sap-hana-gce-overview.json)|
+|This dashboard has charts displaying `Instance Memory`, `Service Memory Used`, `Schema Estimated Max Memory`, `Rowstore Memory Size`, `Schema Read Outliers`,`Schema Write Outlines`, `Schema Record Count Outliers`, `System Connections`, `Alerts`, and `Blocked Transactions` from SAP HANA as well as charts of infrastructure related metrics for the running SAP HANA VMs: `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`. There is also a card with links to docs and the SAP HANA logs in Cloud Logging.|
```
Comment (EvanSimpson) [inline dashboards/sap-hana/README.md:6]:
```suggestion
|This dashboard is based on [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/sap_hana#monitored-metrics) and has charts displaying `Instance Memory`, `Service Memory Used`, `Schema Estimated Max Memory`, `Rowstore Memory Size`, `Schema Read Outliers`,`Schema Write Outlines`, `Schema Record Count Outliers`, `System Connections`, `Alerts`, and `Blocked Transactions` from SAP HANA as well as charts of infrastructure related metrics for the running SAP HANA VMs: `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`. There is also a card with links to docs and the SAP HANA logs in Cloud Logging.|
```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/sap-hana/metadata.yaml (hunk 1 above)
Comment (megelatim) [reply dashboards/sap-hana/metadata.yaml]: done

**Review by EvanSimpson — APPROVED** [review]

File: dashboards/sap-hana/README.md (hunk 2 above)
Comment (megelatim) [reply dashboards/sap-hana/README.md]: done

File: dashboards/sap-hana/README.md (hunk 3 above)
Comment (megelatim) [reply dashboards/sap-hana/README.md]: done

File: dashboards/sap-hana/README.md (hunk 4 above)
Comment (megelatim) [reply dashboards/sap-hana/README.md]: done

**Comment by EvanSimpson** [comment]
Thank you!

**Pull request merged**